flake.nix

```diff
@@ -38,21 +40,65 @@
       };
 
     in {
-      # This flake exposes one attribute: a development shell
-      # containing the rpki-client and the necessary Python env and packages to run kartograf.
-      # To use, run 'nix develop' in the current directory.
+      # This flake exposes the following attributes:
+      # * A development shell containing the rpki-client and the necessary
+      #   Python env and packages to run kartograf. To use, run 'nix develop'
+      #   in the current directory.
+      # * A default/kartograf package 
+      # * A NixOS module
       devShell = pkgs.mkShell {
-          packages = [
+        packages = [
+          rpki-cli.defaultPackage.${system}
+          # Python 3.10 with packages
+          (pkgs.python311.withPackages (ps: [
+            ps.pandas
+            ps.beautifulsoup4
+            ps.requests
+            ps.tqdm
+          ]))
+          pandarallel
+        ];
+      };
+      packages = {
+        kartograf = pkgs.stdenv.mkDerivation { # not a python-installable package, so just manually copy files
+          pname = "kartograf";
+          version = "1.0.0";
+          src = ./.;
+          nativeBuildInputs = [ pkgs.makeWrapper ];
+          buildInputs = with pkgs.python3Packages; [
             rpki-cli.defaultPackage.${system}
-            # Python 3.10 with packages
-            (pkgs.python311.withPackages (ps: [
-              ps.pandas
-              ps.beautifulsoup4
-              ps.requests
-              ps.tqdm
-            ]))
+            pkgs.python3
```

Comment:
nice, i guess the deps gets linked to the right python version without setting it explicitly like at line 53 ? If so could you just use the same package declaration in both places?

Reply:
yep, nice.